server/start.go

```diff
@@ -145,10 +145,6 @@ which accepts a path for the resulting pprof file.
 			}
 
 			serverCtx.Logger.Debug("received quit signal")
-			// TODO: why is this check here? Should not make sense since err is checked above
```

Comment:
why is this check here? doesnt make sense since the error is checked above. can likely remove it